### PR TITLE
[*.py] Rename "Arguments:" to "Args:"

### PR DESCRIPTION
**Pull request by SamuelMarks**
I've written custom parsers and emitters for everything from docstrings to classes and functions. However, I recently came across an issue when I was parsing/generating from the TensorFlow codebase: inconsistent use of `Args:` and `Arguments:` in its docstrings.

```sh
(pytorch#c348fae)$ for name in 'Args:' 'Arguments:'; do
    printf '%-10s %04d\n' "$name" "$(rg -IFtpy --count-matches "$name" | paste -s -d+ -- | bc)"; done
Args:      1095
Arguments: 0336
```

It is easy enough to extend my parsers to support both variants, however it looks like `Arguments:` is wrong anyway, as per:

  - https://google.github.io/styleguide/pyguide.html#doc-function-args @ [`ddccc0f`](https://github.com/google/styleguide/blob/ddccc0f/pyguide.md)

  - https://chromium.googlesource.com/chromiumos/docs/+/master/styleguide/python.md#describing-arguments-in-docstrings @ [`9fc0fc0`](https://chromium.googlesource.com/chromiumos/docs/+/9fc0fc0/styleguide/python.md)

  - https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html @ [`c0ae8e3`](https://github.com/sphinx-contrib/napoleon/blob/c0ae8e3/docs/source/example_google.rst)

Therefore, only `Args:` is valid. This PR replaces them throughout the codebase.

PS: For related PRs, see tensorflow/tensorflow/pull/45420

PPS: The trackbacks automatically appearing below are sending the same changes to other repositories in the [PyTorch](https://github.com/pytorch) organisation.

### CODE REVIEW

File: tools/codegen/model.py [hunk 1]
```diff
@@ -891,7 +891,7 @@ def all(self) -> Sequence[Argument]:
         return [self.dtype, self.layout, self.device, self.pin_memory]
 
 @dataclass(frozen=True)
-class Arguments:
+class Args:
```
Comment (myleott) [inline tools/codegen/model.py:894]: this seems wrong...

Comment (SamuelMarks) [reply tools/codegen/model.py:894]: Ahhh yep, fixed

**Review by soumith — APPROVED** [review]

**Comment by SamuelMarks** [comment]
Great. Not sure why it's red not purple but good to see it merged

**Comment by soumith** [comment]
thanks for the patch. the merge is a little round-about, so the Red is normal.

**Pull request closed without merging**